espnet2/asr/encoder/contextual_block_conformer_encoder.py

```diff
@@ -208,17 +208,19 @@ def forward(
         ilens: torch.Tensor,
         prev_states: torch.Tensor = None,
         is_final=True,
+        infer=False,

```

Comment:
Can you come up with a better variable name?
`infer` is a bit ambiguous.
`infer_mode`?

Reply:
OK! I think "infer_mode" is good.